Automattic/pocket-casts-android
# Added Tasker Actions: Query Podcasts, Query Podcast Episodes, Query Filters, Query Filter Episodes, Add To Up Next

## Description
Added 6 new Tasker Actions that allows users to query data from Pocket Casts and manage the Up Next Queue: 

- Query Podcasts
- Query Podcast Episodes
- Query Filters
- Query Filter Episodes
- Add To Up Next
- Query Up Next Episodes

This allows users to setup their own up next queue manager in Tasker like for example  as shown in these demos: 

- creating a simple Pocket Casts Android Wear app: https://youtu.be/opzeAOuLOjI
- creating a notification that automatically plays your favorite podcasts on the tap of a button: https://youtu.be/ifpf-D8w0b4
- creating a playlist manager: https://youtu.be/aaKggzKK2QM

More output info can be added to the Query actions very easily if needed.

**Note**: If you could test this in a release build and check that there no crashes that would be great :)

## Testing Instructions
1. Open Tasker
2. Add the **Pocket Casts Query Podcasts** action
3. Add a flash action with the variables you want from the above action
4. Add the **Pocket Casts Query Podcast Episodes**
5. Select any podcast you want
6. Again, check that the variables are correctly populated
7. Do the same for **Pocket Casts Query Filters** and **Pocket Casts Query Filter Episodes** actions
8. Add a **Pocket Casts Add To Up Next** action
9. Use the result UUID of a podcast episode as this action's input (for example, to use the first result from a query above use the **%episode_uuid(1)** value
10.  Check if the queue was set correctly in Pocket Casts


## Checklist
 
#### I have tested any UI changes...

- x] with the device set to have a large display and font size
